style.css

```diff
@@ -1,11 +1,12 @@
 @font-face {
   font-family: HanaMinA;
-  src: url(HanaMinA.woff);
+  src: local("HanaMinA")
```

Comment:
The idea is still to offer exactly the font necessary so that everyone sees exactly the same, so let's remove local(), because there's no guarantee their local font is the same font version as the one used to build the woff subset font.

Reply:
323K as woff, 241K as woff2

Reply:
brotli showing off its power. Very nice!